src/executables/unloaddb.h

```diff
@@ -59,6 +59,21 @@ extern int lo_count;
 #define PRINT_IDENTIFIER(s) "[", (s), "]"
 #define PRINT_FUNCTION_INDEX_NAME(s) "\"", (s), "\""
 
+/* 
+ * name is user_specified_name.
+ * owner_name must be a char array of size DB_MAX_IDENTIFIER_LENGTH to copy user_specified_name.
+ * class_name refers to class_name after dot(.).
+ */
+#define SPLIT_USER_SPECIFIED_NAME(name, owner_name, class_name) \
+	do \
+	  { \
+	    memset ((owner_name), '\0', DB_MAX_IDENTIFIER_LENGTH); \
```

Comment:
memset() is unnecessary code.

Reply:
It is checked with unnecessary memset() and will be removed.